Sample/Kitchen Sink/DemonstrationTools/UnitConverter.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  UnitConverter.swift
+//  Kitchen Sink
+//
+//  Created by Jake Dove on 11/03/2024.
+//
+
+import Foundation
+import SwiftUI
+import FlightUI
+
+struct UnitConverter: View {
+    @EnvironmentObject var theme: Theme
+    @StateObject var themeManager = ThemeManager(current: .dark)
+    @ObservedObject var demonstrationVM = DemonstrationViewModel()
```

Comment:
Should the viewmodel have a name related to the file in which its used?
`UnitConvertViewModel.swift`?

Reply:
Agreed. Corrected